util/exercise/src/lib.rs

```diff
@@ -17,16 +33,59 @@ lazy_static! {
             .expect("Failed to get the path to the track repo.");
 
         String::from_utf8(rev_parse_output.stdout)
-            .unwrap()
+            .expect("git rev-parse produced non-utf8 output")
             .trim()
             .to_string()
     };
 }
 
-pub fn run_configlet_command(command: &str, args: &[&str]) {
+#[macro_export]
+macro_rules! val_as {
+    ($value:expr, $as:ident) => {
+        $value
+            .$as()
+            .ok_or_else(|| $crate::errors::Error::SchemaTypeError {
+                file: "config.json".to_string(),
```

Comment:
Since the `val_as!` is called both when parsing canonical-data and config.json, perhaps the hard-coded `file: "config.json".to_string()` line should be changed here?

Reply:
I added cases to the macro to make that an overrideable default in https://github.com/exercism/rust/pull/700/commits/e3c726e4275336c4f3458a8fa133954f81ca1885; it just hasn't yet been merged.